apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -7,16 +7,28 @@ import {
 import { EMPTY_LENGTH } from "~/libs/constants/constants.js";
 import { type TableColumn } from "~/libs/types/types.js";
 
+import { SelectRowCell } from "./components/select-row-cell/select-row-cell.js";
```

Comment:
```suggestion
import { SelectRowCell } from "./components/components.js";
```

Reply:
SelectRowCell is a subcomponent of Table. There is no need to export it. Check [this](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/176#discussion_r1745533946)

Reply:
Yep, I know. I'm about creating `components.ts` inside `table/components` folder

Reply:
Oh, my bad. Thanks for the clarification!